care/facility/models/__init__.py

```diff
@@ -4,7 +4,6 @@
 from .ambulance import *  # noqa
 from .asset import *  # noqa
 from .bed import *  # noqa
-from .camera_preset import *  # noqa
```

Comment:
removing this now will cause a pending make migration as 0 references are there to the `CameraPreset` model now.

<img width="644" alt="image" src="https://github.com/user-attachments/assets/96981c0b-c90b-4e7a-a83d-8eee15d3d408">